docs/ble.md

```diff
@@ -25,21 +38,22 @@ treat them like decimals.*
 Web IDL
 -------
 This IDL provides an overview of the interface; see below for documentation of
-specific API functions.
+specific API functions.  We also have a short document explaining [ZJS WebIDL conventions](Notes_on_WebIDL.md).
+
 
 ```javascript
 // require returns a BLE object
 // var ble = require('ble');
 
-[NoInterfaceObject]
+[ReturnFromRequire]
 interface BLE: EventEmitter {
     void disconnect(string address);
     void startAdvertising(string name, string[] uuids, string url);
     void stopAdvertising();
-    void setServices(PrimaryService services[]);
-    PrimaryService PrimaryService(PrimaryServiceInit init);
-    Characteristic Characteristic(CharacteristicInit init);
-    Descriptor Descriptor(DescriptorInit init);
+    void setServices(PrimaryService[] services);
```

Comment:
in Web IDL we don't use [], you will normally use `sequence<PrimaryService>` or `FrozenArray<PrimaryService>` depending on the exact use-case.